storage/innobase/xtrabackup/test/t/xb_export_check_cfg.sh

```diff
@@ -0,0 +1,43 @@
+. inc/common.sh
+
+start_server 
+
+vlog "check if cfg version is 6 in table"
+
+mkfifo $topdir/fifo
+
+$MYSQL $MYSQL_ARGS <$topdir/fifo &
+
+client_pid=$!
+
+# Open the pipe for writing. This is required to prevent cat from closing the
+# pipe when stdout is redirected to it
+
+exec 3>$topdir/fifo
+
+cat >&3 <<EOF
+USE test;
+CREATE TABLE t1(a INT) ENGINE=InnoDB;
+FLUSH TABLE t1 FOR EXPORT;
+EOF
+
+
+# Let the client complete the above set of statements
+vlog "waiting for 3 seconds to ensure cfg file is generated"
+sleep 3
+
+CFG_VERSION=`od -j3 -N1 -t x1 -An $mysql_datadir/test/t1.cfg  | head -30 | tr -d ' ' | tr -d '\n'`
+
+if [ "$CFG_VERSION" = "06" ]; then
+	vlog "correct version of CFG"
+else
+	vlog "wrong version of CFG $CFG_VERSION"
+	exit -1
+fi
+
+# Terminate the background client
+echo "exit" >&3
+exec 3>&-
+wait $client_pid
+
```

Comment:
why do we need a background client at all? connection disconnection will delete the .cfg file?

Reply:
yeah  disconnection will unlock table delete the cfg file.